### ISSUE TITLE
Support `pre=True` in pydantic validators

### ISSUE THREAD

**Issue by dsinghvi** [fern-api/fern-python#132]
The API for registering types will look like 

```python
# field validators
@MyType.Validators.field("abc", pre=True)
def validateAbc(): 
 ...

# root validators
@MyType.Validators.root(pre=True)
def validate(): 
 ...
```

**Comment by dsinghvi** [comment]
https://github.com/fern-api/fern-python/pull/134 support `pre` for root validators

**Comment by dsinghvi** [comment]
Released as part of [0.0.27](https://github.com/fern-api/fern-python/releases/tag/0.0.27)
